tests/cont-scripts/fail-both-price-up-01-cont.yaml

```diff
@@ -1,6 +1,9 @@
 # Both debtor and creditor can finish together, but cannot negotiate price up
 type: "cont"
-pactId: 5
+pactId:
```

Comment:
Yeah, seeing as you have to run something to see the hash, we might as well just have to find the pact id and paste it in.

Reply:
Gotcha so just have this be a Hash value instead of `ApiPactId`?

Reply:
Hashes make sense here.

Reply:
See https://github.com/kadena-io/pact/pull/475#discussion_r279093324